### DISCUSSION TITLE
Examples of template components

### DISCUSSION THREAD

**Discussion by afidegnum** [framesurge/perseus#144] [Ideas]
Can you please add to the examples how to handles template parts (Components) [similar to Sycamore's ](https://github.com/sycamore-rs/sycamore/blob/0.7.1/examples/components/src/main.rs)

**Comment by lukechu10** [comment]
Unless you are looking for https://github.com/arctic-hen7/perseus/issues/4, you can just use plain old sycamore components inside your perseus app. Sycamore components are just like function calls, so it behaves the same as if you just inlined your entire component into the call site.
